tasks/Debian.yml

```diff
@@ -7,9 +7,10 @@
     - "linux-image-extra-virtual-lts-{{ ansible_distribution_release }}"
```

Comment:
Packages to go are: "linux-image-extra-virtual", "linux-image-extra-virtual-lts-{{ ansible_distribution_release }}", "python", "python-pip", "python-httplib2". None of these are listed as dependencies in https://docs.docker.com/engine/installation/linux/docker-ce/ubuntu/#install-using-the-repository.